src/main/java/co/omise/models/Refund.java

```diff
@@ -169,7 +198,8 @@ protected HttpUrl path() {
 
         @Override
         protected ResponseType<ScopedList<Refund>> type() {
-            return new ResponseType<>(new TypeReference<ScopedList<Refund>>() {});
+            return new ResponseType<>(new TypeReference<ScopedList<Refund>>() {
```

Comment:
Should this be the same line?

Reply:
That's how the IDE cleans it up, I've no preference